pkg/controller/controller.go

```diff
@@ -33,3 +37,20 @@ func AddToManager(m manager.Manager) error {
 
 	return nil
 }
+
+func getActuatorParams(mgr manager.Manager) osactuators.ActuatorParams {
```

Comment:
Stutter (`actuators.Actuator`)
```suggestion
func getActuatorParams(mgr manager.Manager) openstack.ActuatorParams {
```